jupyterlab_nodeeditor/yggdrasil_support.py

```diff
@@ -1,9 +1,37 @@
 import uuid
-
 import yaml
 
 from .node_editor import NodeEditor
 
+from yggdrasil.examples import yamls as ex_yamls
+import jupyterlab_nodeeditor as jlne
+import yggdrasil.yamlfile
+import re
+
+photosynthesis_model = {}
+with open(ex_yamls['fakeplant']['python'], "r") as test_model:
+    photosynthesis_model = yaml.safe_load(test_model)['model']
+    
+# Hard coded function to clean up Photosynthesis model
+def dict_conversion(model_dict):
```

Comment:
I think I would like it if we changed this to return a new version.  That would mean that we accept `model_dict` as input, and then everywhere you've got a `.pop()` that doesn't get assigned, you could skip that line.

Reply:
Fixed everything else locally, but just to clarify on this, you mean return a new dict right? Instead of modifying the input and that way we can just pull and copy data instead of deleting existing